$NetBSD: patch-configure.ac,v 1.2 2023/05/22 16:40:37 schmonz Exp $

* For newer Autotools
* Detect Python version in another way
* Find Tcl in any pkgsrc PREFIX

--- configure.ac.orig	2013-08-06 15:46:42.000000000 +0000
+++ configure.ac
@@ -1,4 +1,4 @@
-AC_INIT([newt_pr.h])
+AC_INIT([newt_pr.h], 0.52.16)
 
 PACKAGE=newt
 VERSION=$(awk '/^Version: / {print $2}' $srcdir/newt.spec)
@@ -60,7 +60,8 @@ if test "x$with_python" = "xno"; then
   AC_MSG_RESULT([skipped])
   PYTHONVERS=
 else
-  PYTHONVERS=$(ls /usr/include/python2.*/Python.h 2> /dev/null | sed 's|/usr/include/\([[^/]]*\)/Python.h|\1|g' | tr '\n' ' ')
+  AM_PATH_PYTHON([2.5],, [:])
+  PYTHONVERS=${PYTHON_VERSION}
   AC_MSG_RESULT([$PYTHONVERS])
 fi
 AC_SUBST([PYTHONVERS])
@@ -78,7 +79,7 @@ else
       `ls -d /usr/lib64 2>/dev/null` \
       `ls -d /usr/local/lib 2>/dev/null` \
       `ls -d /usr/contrib/lib 2>/dev/null` \
-      `ls -d /usr/pkg/lib 2>/dev/null` \
+      `ls -d @PREFIX@/lib 2>/dev/null` \
       `ls -d /usr/local/lib/tcl8.5 2>/dev/null` \
       `ls -d /usr/local/lib/tcl8.4 2>/dev/null` \
       `ls -d /usr/lib/tcl8.5 2>/dev/null` \
